### PR TITLE
✨ add new docs

### PR DESCRIPTION
**Pull request by hdsujnb**
Added new docs for backend.todorant.com.

The documentation was created using mkdocs with the Material theme. The work of the documentation has been tested.

Documentation generation requires Python >=3.6.

Added 2 dependencies:
- koa-static
- koa-mount

Added a generate-docs script to generate documentation. It runs in the `distribute` script

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -13,6 +13,8 @@ import '@/helpers/googleCalendarChannel'
 import '@/sockets'
 import 'reflect-metadata'
 import * as Koa from 'koa'
+import serve = require('koa-static')
```
Comment (backmeupplz) [inline src/app.ts:16]: Can we use esm imports here?

File: src/controllers/public.ts [hunk 2]
```diff
@@ -10,12 +9,8 @@ export default class PublicController {
...
   docs(@Ctx() ctx: Context) {
     try {
-      const converter = new showdown.Converter()
-      const md = readFileSync(`${__dirname}/../../docs/README.md`, 'utf8')
-      const css = readFileSync(`${__dirname}/../../docs/style.css`)
-      return `<style>${css}</style><div class="markdown-body">${converter.makeHtml(
-        md
-      )}</div>`
+      ctx.redirect("docs/")
+      return 200
```
Comment (backmeupplz) [inline src/controllers/public.ts:13]: Let's do `permanent redirect` status

File: package.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
   "license": "MPL-2.0",
   "private": true,
   "scripts": {
-    "distribute": "yarn build-ts && node dist/server.js",
+    "distribute": "yarn build-ts && yarn generate-docs && node dist/server.js",
```
Comment (backmeupplz) [inline package.json:11]: Will this work after `cd` in `generate-docs`?

File: package.json [hunk 4]
```diff
@@ -8,7 +8,7 @@
   "license": "MPL-2.0",
```
Comment (backmeupplz) [inline package.json:8]: Let's not use `npm`, please, get rid of `package-lock.json` :)

**Review by backmeupplz — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (hdsujnb) [reply package.json:11]: Yes, it works. (I just checked.)

**Pull request merged**